.github/workflows/deploy.yml

```diff
@@ -15,6 +15,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
+    - run: git checkout HEAD
+    - run: git fetch origin master:master
```

Comment:
I don't understand why we have to do this manually. Let's 📞

Reply:
Because the checkout github action only checkout the current branch and we need to have the current branch and the master to allow getting all files that are modified between these two branches.
I though with the option `fetch-depth: 0` it should not be necessary as explained [here](https://github.com/actions/checkout#Checkout-a-different-branch), but if I remove `git checkout HEAD` and `git fetch origin master:master` it does not work… Not very clear for me.